apcd-cms/src/apps/admin_exception/templates/list_admin_exception.html

```diff
@@ -2,22 +2,49 @@
 {% load static %}
 
 {% block content %}
-
 <link rel="stylesheet" href="{% static 'apcd-cms/css/table.css' %}">
 <link rel="stylesheet" href="{% static 'apcd-cms/css/modal.css' %}">
-<link rel="stylesheet" href="{% static 'submissions/css/table.css' %}">
+<!-- Must come before submissions modal stylesheet so key names in modal are not cut off -->
+<link rel="stylesheet" href="{% static 'admin_exception/css/table.css' %}">
 <link rel="stylesheet" href="{% static 'submissions/css/modal.css' %}">
-<link rel="stylesheet" href="{% static 'admin_submissions/css/admin_table.css' %}">
 <link rel="stylesheet" href="{% static 'admin_exception/css/modal.css' %}">
-
 <div class="container">
   {% include "nav_cms_breadcrumbs.html" %}
 
   <h1>View Exception Requests</h1>
   <hr />
   <p style="margin-bottom: 30px">All submitted exception requests</p>
-  <hr />
-  <table id="submissionTable" class="submission-table">
+  <div class="filter-container">
+    <div class="filter-content">
+      <span><b>Filter by Status: </b></span>
+      <select id="statusFilter" class="status-filter" onchange="filterTableByStatus()">
+        {% for option in status_options %}
+          <option class="dropdown-text" {% if option == selected_status %}selected{% endif %}>{{ option }}</option>
+        {% endfor %}
+      </select>
+      </div>
+      </div>
+      <div class="filter-container">
+      <div class="filter-content">
+      <span><b>Filter by Organization: </b></span>
+      <select id="organizationFilter" class="status-filter org-filter" onchange="sortByOrg()">
+        <!-- Value set here so dropdown is not blank but set to All-->
+        <option class="dropdown-text" value={{selected_org.all}} {% if not selected_org %}selected{% else %} hidden {% endif %} disabled></option>
+        {% for option in org_options %}
+          <option class="dropdown-text" {% if option == selected_org %}selected{% endif %}>{{ option }}</option>
+        {% endfor %}
+      </select>
+      </div>
+      </div>
+      {% if selected_status or selected_org %}
+      <div class="filter-container">
+        <div class="filter-content">
+          <button onclick="clearSelections()">Clear Options</button>
+        </div>
+      </div>
+      {% endif %}
+  <hr/>
```

Comment:
I suggest moving this `<hr>` element to before the filter container to align with design on other listing pages.

Reply:
Good idea! Made the filters more in line with Wes' styles and moved the hr to match other pages